### DISCUSSION TITLE
Encryption

### DISCUSSION THREAD

**Discussion by sreejesh123** [strimzi/strimzi-kafka-operator#5324] [Q&A] [answered]
Hi,  by default when I setup Strimzi vanilla installation,   operator , kafka+ZK   .. will all the inter POD communication between

a. between  broker nodes
b. between ZK  nodes
c. ZK to broker nodes

will it all be encrypted?   is this config controlled to Enable to Disable encrypted comm between internal node?

Sorry for lame Q , as we use vanilla Apache  kafka, and this is not encrypted and hence wanted to see by default what setup is provided by Strimzi?

**Comment by scholzj** [comment, marked as answer]
Yes, all the internal communication is encrypted out of the box. It also uses TLS client authentication, not just encryption.

**Reply by sreejesh123** [reply]
thank you, to provide for my internal approval and control signoff to use strimzi. possible to point me to the document which talks about it at all please?

**Reply by scholzj** [reply]
I guess this talks about it in the first paragraph? https://strimzi.io/docs/operators/latest/full/using.html#security-str
